src/main/java/io/getunleash/repository/OkHttpFeatureFetcher.java

```diff
@@ -16,11 +16,15 @@
 import okhttp3.Request;
 import okhttp3.Response;
 
+import static io.getunleash.util.UnleashConfig.UNLEASH_INTERVAL;
+
 public class OkHttpFeatureFetcher implements FeatureFetcher {
     private final HttpUrl toggleUrl;
     private final OkHttpClient client;
+    private final UnleashConfig config;
```

Comment:
do you need to store the entire config if you only need the interval?

Reply:
Storing interval only now